include/bluetooth/mesh/sensor_cli.h

```diff
@@ -282,10 +289,10 @@ struct bt_mesh_sensor_cli_handlers {
  *                         not configured.
  *  @retval -EAGAIN        The device has not been provisioned.
  */
-int bt_mesh_sensor_cli_list_get(struct bt_mesh_sensor_cli *cli,
-				struct bt_mesh_msg_ctx *ctx,
-				struct bt_mesh_sensor_info *sensors,
-				uint32_t *count);
+int bt_mesh_sensor_cli_desc_all_get(struct bt_mesh_sensor_cli *cli,
+				    struct bt_mesh_msg_ctx *ctx,
+				    struct bt_mesh_sensor_info *sensors,
+				    uint32_t *count);
```

Comment:
CI complains about a `@ref` to this function on line 146 that needs to be updated to the new name